ckanext/who_afro/templates/sources/index.html

```diff
@@ -1,203 +1,210 @@
-{% extends "page.html" %} {% block subtitle %} {{ _('Data Sources') }}
-{%endblock subtitle%} {% block breadcrumb_content %}
-<li class="active">
-  {% link_for _('Data Sources'), named_route='sources.index' %}
-</li>
-{% endblock %} {% block content %}
-<div role="main" class="insights-page">
-  <div class="promoted-background">
-    {% snippet 'package/snippets/promoted_plus.html' %}
-    <div class="container">
-      <div class="promoted promoted-home col-xs-12 col-sm-10 col-md-8 col-lg-6">
-        <div class="promoted-container">
-          <p class="subtitle">{{ _('Home') }} /</p>
-          <h1 class="page-title">{{ _('Data Sources') }}</h1>
-          <p>
-            {{ _('The WHO African Regional Health Data Hub integrates public
-            health and related data from across the region and beyond. We draw
-            on a variety of sources including the following data sources.') }}
-          </p>
+{% extends "page.html" %}
+{% block subtitle %}
+    {{ _("Data Sources") }}
+{% endblock subtitle %}
+
+{% block breadcrumb_content %}
+    <div class="active">{% link_for _('Data Sources'), named_route='sources.index' %}</div>
+{% endblock %}
+
+{% block content %}
+    <div role="main" class="sources-page">
+        <div class="promoted-background">
+            {% snippet 'package/snippets/promoted_plus.html' %}
+            <div class="container">
+                <div class="promoted promoted-home col-xs-12 col-sm-10 col-md-8 col-lg-6">
+                    <div class="promoted-container">
+                        <p class="subtitle">{{ _("Home") }} /</p>
+                        <h1 class="page-title">{{ _("Data Sources") }}</h1>
+                        <p>
+                            {{ _('The WHO African Regional Health Data Hub integrates pubdivc
+                                  health and related data from across the region and beyond. We draw
+                                  on a variety of sources including the following data sources.') }}
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="col-1 text-center">
```

Comment:
I was thinking something like `col-lg-1 col-sm-12` in order to take into account mobile devices. This is obviously just a first step because further styling needs to be applied to the image and the text to obtain something looking good.

If you try this on a tablet or mobile, it won't look.
An initial attempt on side ends up with something like the following image:

![Screenshot from 2024-08-20 14-10-17](https://github.com/user-attachments/assets/1f480f85-5e75-48a6-9397-e2ef0ae50aa3)

(If you're busy with other stuff, I can work on this as part of my current work to make the site responsive on smaller devices and just merge this PR.)

Reply:
To clarify, the image above is when I modify your work as an attempt to show what we probably want on a mobile device.
If you try the current code, it doesn't look good.

Reply:
Absolutely, I'm in agreement with you: had we had more time, Lewis would have done the design for mobile and tablets as well.
Nonetheless, if my memory serves me right, the client wanted a responsive design so the big idea with my comment is to try to make things responsive in limit of the tools at our disposal until Lewis can do the **proper** mobile designs.

Reply:
Go for it then :smile: add a commit to this branch with your solution please.

Reply:
I've made this table properly responsive now. All should be good.

Reply:
:pray: thanks guys for correcting my dodgy work :rofl: